lab_1_classify_by_unigrams/main.py

```diff
@@ -11,6 +13,14 @@ def tokenize(text: str) -> list[str] | None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
+    if isinstance(text, str):
```

Comment:
Let's try to use early return: https://medium.com/swlh/return-early-pattern-3d18a41bba8

Reply:
done